### ISSUE TITLE
Release 0.4.5 Error: cannot get default rules: Error: failed to get ruleset java-best-practices

### ISSUE THREAD

**Issue by e2e-tests-prd** [DataDog/datadog-static-analyzer#540]
**Describe the bug**
Release 0.4.5 is failing on getting the ruleset java-best-practices

**To Reproduce**

Add this stage to `.gitlab-ci.yml` in a node application.
```
dd_sast:
  stage: sast
  image: node:20
  variables:
    DATADOG_STATIC_ANALYZER_URL: "https://github.com/DataDog/datadog-static-analyzer/releases/latest/download/datadog-static-analyzer-x86_64-unknown-linux-gnu.zip"
  script:
    - echo "Installing Datadog Static Analyzer"
    - npm install -g @datadog/datadog-ci
    - curl -L -o /tmp/ddog-static-analyzer.zip $DATADOG_STATIC_ANALYZER_URL
    - unzip /tmp/ddog-static-analyzer.zip -d /tmp
    - mv /tmp/datadog-static-analyzer /usr/local/datadog-static-analyzer
    - OUTPUT_FILE="/tmp/report.sarif"
    - echo "Will output to $OUTPUT_FILE"
    - /usr/local/datadog-static-analyzer -i "./path/to/your/project" -o "$OUTPUT_FILE" -f sarif
    - echo "Uploading results to Datadog"
    - datadog-ci sarif upload --service "$DD_SERVICE_NAME" --env "$ENVIRONMENT" "$OUTPUT_FILE"
    - cp "$OUTPUT_FILE" .
  artifacts:
    paths:
      - /tmp/report.sarif
    expire_in: 1 hour
  rules:
    - if: '$CI_COMMIT_BRANCH'
      when: on_success
```

**Expected behavior**
Expected to not have an error thrown (Error: failed to get ruleset java-best-practices) on getting a ruleset that was not defined. Version 0.4.4 works as expected.

**Desktop (please complete the following information):**
 - OS: node:20
 - Browser CI
 - Version 0.4.5

**Smartphone (please complete the following information):**

**Additional context**
